browser.go

```diff
@@ -214,11 +220,11 @@ func (browser *Browser) IsAndroid() bool {
 }
 
 func (browser *Browser) IsIPhone() bool {
-	return browser.Platform == "iOS" && browser.DeviceName == "iPhone"
+	return browser.Platform == "iOS" && browser.DeviceCodeName == "iPhone"
 }
 
 func (browser *Browser) IsIPad() bool {
-	return browser.Platform == "iOS" && browser.DeviceName == "iPad"
+	return browser.Platform == "iOS" && browser.DeviceCodeName == "iPad"
```

Comment:
typical example:
Device_Name="Galaxy S5 LTE"
Device_Code_Name="SM-G900F"

Shouldn't this line still be:
return browser.Platform == "iOS" && browser.DeviceName == "iPad"